newsfragments/config.toml

```diff
@@ -16,7 +16,7 @@
 # under the License.
 [tool.towncrier]
 name = "Airflow"
-filename = "../RELEASE_NOTES.rst"
+filename = "RELEASE_NOTES.rst"
```

Comment:
I don't see the change in the RELEASE_NOTES.rst doc path. Any reason why its path changed here?

Reply:
I guess it's the matter where you run towncrier (my guess).

Reply:
Yeah, that's exactly it.

Reply:
This allows the towncrier step [here](https://github.com/apache/airflow/blob/main/dev/README_RELEASE_AIRFLOW.md) to work from the repo root.